internal/kafka/internal/handlers/validation.go

```diff
@@ -37,9 +37,22 @@ const minimunNumberOfNodesForTheKafkaMachinePool = 3
 
 func validateKafkaBillingModel(ctx context.Context, kafkaService services.KafkaService, kafkaConfig *config.KafkaConfig, kafkaRequestPayload *public.KafkaRequestPayload) handlers.Validate {
 	return func() *errors.ServiceError {
+		billingModel := shared.SafeString(kafkaRequestPayload.BillingModel)
+		// enterprise kafkas billing model validation
+
+		if !shared.StringEmpty(kafkaRequestPayload.ClusterId) && !shared.StringEqualsIgnoreCase(billingModel, constants.BillingModelEnterprise.String()) {
+			return errors.InvalidBillingAccount("invalid billing model: %s, only %v is allowed", billingModel,
```

Comment:
can be `%s` since `.String()` is called on the billing model. Or simply always use `%v`.

Reply:
+1, I'll change it.

Reply:
done